### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by gdanov** [CleverTap/clevertap-react-native#307]
Can't build **locally** (previously running) project after changing laptop. The project was just copied. Cloud build is OK

This is the error
```
[RUN_FASTLANE] › Compiling clevertap-react-native Pods/CleverTap-iOS-SDK » CTLocalDataStore.m
[RUN_FASTLANE]
❌  (ios/Pods/CleverTap-iOS-SDK/CleverTapSDK/CTLocalDataStore.m:416:1)

  414 |         return nil;
  415 |     }
> 416 | }
      | ^ non-void function does not return a value in all control paths [-Werror,-Wreturn-type]
  417 |
  418 |
  419 | #pragma mark - Public API
```

build env

```
[RUN_FASTLANE] +-------------+-------------------------+
[RUN_FASTLANE] |           Build environment           |
[RUN_FASTLANE] +-------------+-------------------------+
[RUN_FASTLANE] | xcode_path  | /Applications/Xcode.app |
[RUN_FASTLANE] | gym_version | 2.212.1                 |
[RUN_FASTLANE] | sdk         | iPhoneOS16.4.sdk        |
[RUN_FASTLANE] +-------------+-------------------------+
```

expo latest version, clevertap `"clevertap-react-native": "^1.0.1",`

**Comment by akashvercetti** [comment]
@gdanov we are aware of this issue steming from Xcode 14.3 and above. We will be releasing a fix in this week itself. in the meantime, you can edit your fastlane or your CICD pipeline yaml to use XCode 14.2 in the meantime.

**Comment by gdanov** [comment]
Thanks! how do I set this? Googling didn't help

**Comment by sneh-lio** [comment]
Hey @akashvercetti 
How can I fix this on my local machine? Do you have any quick fix?

**Comment by gdanov** [comment]
fastlane's `xcversion` seems to be the proper keyword

putting `.xcode-version` file with content `14.2` in your ios folder did the trick for me. Must remember to remove it at some point ...

**Comment by akashvercetti** [comment]
We have added a fix for this in our latest release v1.0.2. https://github.com/CleverTap/clevertap-react-native/releases/tag/1.0.2 
Could you please try it out?
